changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Subscriptions Core Changelog ***
 
+= 5.3.0 - 2023-xx-xx =
+* Fix - Use shipping address at checkout when renewing, resubscribing or switching subscriptions that contain different billing and shipping addresses.
```

Comment:
Nitpick: Doesn't fix renewal, could just remove that one word. However, it could cause confusion.
```suggestion
* Fix - Shipping address correctly set when resubscribing or switching subscriptions that contain different billing and shipping addresses.
```